src/v/cloud_storage/remote_partition.cc

```diff
@@ -778,7 +778,8 @@ uint64_t remote_partition::cloud_log_size() const {
 
 ss::future<> remote_partition::serialize_json_manifest_to_output_stream(
   ss::output_stream<char>& output) const {
-    return _manifest_view->stm_manifest().serialize_json(output);
+    auto tmp = _manifest_view->stm_manifest().clone();
+    return tmp.serialize_json(output);
```

Comment:
You have to keep `tmp` alive as the serialisation is async. From the debug build
```
test_id:    rptest.tests.shadow_indexing_admin_api_test.SIAdminApiTest.test_manifest_dump
status:     FAIL
run time:   18.062 seconds


NodeCrash docker-rp-15: ==5293==ERROR: AddressSanitizer: stack-use-after-return on address 0x7f0bcd5bf070 at pc 0x7f0bf8b6878e bp 0x7f0bce6cd080 sp 0x7f0bce6cd078
```

Reply:
fixed that, thanks for the hint